instrumentation/micronaut-1.0/build.gradle.kts

```diff
@@ -21,28 +20,32 @@ dependencies {
     testAnnotationProcessor("io.micronaut:micronaut-inject-java:${micronautVersion}")
 }
 
-/**
- * Todo this does not run tests
- */
-for (version in listOf(micronautVersion, micronaut2Version)) {
+val micronaut2Version = "2.2.3"
+
+for (version in listOf(micronaut2Version)) {
     val versionedConfiguration = configurations.create("test_${version}") {
         extendsFrom(configurations.testRuntimeClasspath.get())
     }
     dependencies {
         versionedConfiguration(project(":instrumentation:netty:netty-4.1"))
         versionedConfiguration("io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-netty-4.1:${versions["opentelemetry_java_agent"]}")
         versionedConfiguration(project(":testing-common"))
-        versionedConfiguration("io.micronaut:micronaut-inject-java:${version}")
         versionedConfiguration("io.micronaut.test:micronaut-test-junit5:${micronautTestVersion}")
+        versionedConfiguration("io.micronaut:micronaut-inject-java:${version}")
```

Comment:
Is this needed? The standard test only uses this as an annotation processor, which means it's not present at runtime for those tests (and thus shouldn't be when switching versions)

Reply:
it seems redundant, removing